tasks/cat3.yml

```diff
@@ -137,7 +137,7 @@
       - audit
 
 - name: "LOW | V-38453 | PATCH | The system package management tool must verify group-ownership on all files and directories associated with packages."
-  command: rpm --setugids {{ item.stdout }} && rpm --setperms {{ item.stdout }}
+  command: rpm --setugids {{ item.stdout }}
```

Comment:
Rather than removing the `--setperms`, which is needed required by the STIG, change this to a shell task.